test/read_mixin_test.rb

```diff
@@ -40,6 +40,24 @@
       assert_equal @post_records, Post.find_by_sql('SELECT * FROM `posts` WHERE (id = 1)')
     end
 
+    it "sends an ActiveSupport::Notification" do
+      notification_received = false
+      record_count = 0
+      class_name = ""
+      ActiveSupport::Notifications.subscribe "instantiation.active_record" do |*args|
+        event = ActiveSupport::Notifications::Event.new(*args)
+        record_count = event.payload[:record_count]
+        class_name = event.payload[:class_name]
+        notification_received = true
+      end
+
+      Kasket.cache.write("#{Post.kasket_key_prefix}id=1", @post_database_result)
+      Post.find_by_sql('SELECT * FROM `posts` WHERE (id = 1)')
+      assert notification_received
+      assert_equal 1, record_count
+      assert_equal "Post", class_name
+    end
+
```

Comment:
It  may also be worth adding a test that the payload is identical to that emitted by rails (in case it has changed in subsequent rails versions)